client/src/components/NotesList/NotesList.jsx

```diff
@@ -1,12 +1,24 @@
 import style from "./NotesList.module.css";
 
-export function NotesList({ noteListData }) {
+export function NotesList({
+  noteListData,
+  handleCreateNewNote,
+  handleNoteSelection,
+}) {
+  console.log(noteListData);
   return (
     <div className={style.notesList}>
       <ul>
+        <li className={style.noteElement} onClick={handleCreateNewNote}>
+          Create a New Note
+        </li>
         {noteListData.map((note, index) => {
           return (
-            <li key={index}>
+            <li
+              key={index}
+              className={style.noteElement}
+              onClick={() => handleNoteSelection(note.id)}
```

Comment:
will need to switch this so that noteList has a column for noteId that points to the id of the note and not use the id from the notelist. so it should be note.noteId instead of note.id

Reply:
changed it to use note.id